core/client/tests/acceptance/settings/general-test.js

```diff
@@ -0,0 +1,161 @@
+/* jshint expr:true */
+import {
+    describe,
+    it,
+    beforeEach,
+    afterEach
+} from 'mocha';
+import { expect } from 'chai';
+import Ember from 'ember';
+import startApp from '../../helpers/start-app';
+import destroyApp from '../../helpers/destroy-app';
+import { invalidateSession, authenticateSession } from 'ghost/tests/helpers/ember-simple-auth';
+
+const {run} = Ember;
+
+describe('Acceptance: Settings - General', function () {
+    let application;
+
+    beforeEach(function() {
+        application = startApp();
+    });
+
+    afterEach(function() {
+        destroyApp(application);
+    });
+
+    it('redirects to signin when not authenticated', function () {
+        invalidateSession(application);
+        visit('/settings/general');
+
+        andThen(function() {
+            expect(currentURL(), 'currentURL').to.equal('/signin');
+        });
+    });
+
+    it('redirects to team page when authenticated as author', function () {
+        let role = server.create('role', {name: 'Author'});
+        let user = server.create('user', {roles: [role], slug: 'test-user'});
+
+        authenticateSession(application);
+        visit('/settings/general');
+
+        andThen(() => {
+            expect(currentURL(), 'currentURL').to.equal('/team/test-user');
+        });
+    });
+
+    it('redirects to team page when authenticated as author', function () {
```

Comment:
_Really_ minor thing, could you change `author` to `editor` here so the description matches the test?